marshmallow_sqlalchemy/convert.py

```diff
@@ -57,8 +57,16 @@ def fields_for_model(self, model, include_fk=False, fields=None):
             if fields and prop.key not in fields:
                 continue
             if hasattr(prop, 'columns'):
-                if not include_fk and prop.columns[0].foreign_keys:
-                    continue
+                if not include_fk:
+                    # Only skip a column if there is no overriden column
+                    # which does not have a Foreign Key.
+                    all_fk = True
```

Comment:
This can be refactored using an `else` clause on the loop:

```python

# Only skip a column if there is no overridden column
# which does not have a Foreign Key.
for column in prop.columns:
    if not column.foreign_keys:
        break
else:
    continue
```